logic/src/commonMain/kotlin/com/wire/kalium/logic/data/featureConfig/FeatureConfigMapper.kt

```diff
@@ -75,6 +82,24 @@ class FeatureConfigMapperImpl : FeatureConfigMapper {
             Status.DISABLED
         )
 
+    @Suppress("MagicNumber")
+    override fun fromDTO(data: FeatureConfigData.MLSMigration?): MLSMigrationModel =
+        data?.let {
+            MLSMigrationModel(
+                it.config.startTime,
+                it.config.finaliseRegardlessAfter,
+                it.config.usersThreshold,
+                it.config.clientsThreshold,
+                fromDTO(it.status)
+            )
+        } ?: MLSMigrationModel(
+            Instant.DISTANT_FUTURE,
+            Instant.DISTANT_FUTURE,
+            100,
```

Comment:
can we have these numbers defined somewhere?

Reply:
Yes or maybe I remove the default value make this function return an optional `MLSMigrationModel `

Reply:
I removed them.